plugins/woocommerce/client/admin/client/settings-payments/components/offline-gateway-list-item/offline-payment-gateway-list-item.tsx

```diff
@@ -55,6 +55,9 @@ export const OfflinePaymentGatewayListItem = ( {
 							settingsUrl={
 								gateway.management._links.settings.href
 							}
+							onboardUrl={
```

Comment:
do we need that for offline item?

Reply:
For consistency, yes. It is a gateway that may require onboarding. It will point to the settings page.